README.md

```diff
@@ -225,7 +228,7 @@ module "eks_main" {
 | loki\_distributor\_limits\_cpu | resources config for kubernetes pod | `string` | `null` | no |
 | loki\_distributor\_limits\_memory | resources config for kubernetes pod | `string` | `null` | no |
 | loki\_distributor\_max\_replicas | loki distributor hpa max replicas | `int` | `1` | no |
-| loki\_querier\_node\_selector | Loki querier nodeSelector | `map{}` | `null` | no | 
```

Comment:
@qedgardo new inputs are missing

Reply:
The new inputs are attributes of the custom_node_groups object, defined on variables.tf